lib/widgets/qpi/buttons.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class LongButton extends StatefulWidget {
+  final String text;
+  final Color buttonColor;
+  final Color textColor;
+LongButton(this.text, this.buttonColor, this.textColor); 
```

Comment:
for the `onPressed`, you can add another variable here

`final VoidCallback onPressed` and add it in the parameter list
source [here](https://www.digitalocean.com/community/tutorials/flutter-widget-communication)

to add it in the page itself, you can do:
```dart
// ...
children: [
  LongButton(text: 'hello', buttonColor: Colors.white, textColor: Colors.black,
    onPressed: () {
      print('hello world');
    },
  ),
// ...
```
i haven't tested it yet but it *should technically* work - let me know if something's wrong